code/modules/mob/observer/eye/eye.dm

```diff
@@ -14,7 +14,10 @@
 	var/acceleration = 1
 	var/owner_follows_eye = 0
 	var/living_eye = TRUE 	// Whether or not the eye uses normal living vision handling.
-
+	
+	var/use_click_handler = FALSE // Set if the eye uses special click handling. Distinct from parent mob click handling for AI etc.
+	var/click_handler_type = /datum/click_handler/eye
```

Comment:
Why just not set `click_hander_type` to `null` and check for that instead of having an additional var, `use_click_handler`?